0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 
-#### Routing complexity should be in the connectors.
+Connectors *peer* with one another to exchange information used to determine the best route for a payment.
 
-Routing is a complex and difficult problem, and ought to be performed by the connectors, not the end users of the system. An important objective is to isolate user software from changes caused by the inevitable evolution of the Interledger routing architecture.
+**Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
 
-#### The system must tolerate wide ledger variation.
+### The Interledger
 
-A basic objective of the Interledger design is to tolerate a wide range of ledger characteristics --- e.g. throughput, latency, fees, access restrictions, reliability, decentralization. In addition it must support all types of different asset characteristics --- e.g. divisibility, fungibility, interest/demurrage, issuance.
+The Interledger protocol suite may be used to transact across any ledgers and connectors, whether they are public or private. There is no single network that all parties must connect to to use these protocols.
 
-### Interledger Protocol Suite
+"The Interledger" is a conceptual network made up of independent and diverse ledgers linked by connectors. Each account "on the Interledger" is part of a particular ledger, but they may transact with others by sending interledger payments through different ledgers and connectors. Like "the Internet", the Interledger is not a single network but is comprised of multiple interconnected networks.
 
-The Interledger architecture is separated into four layers:
+## Interledger Security
 
-![Interledger architecture layers](../shared/graphs/interledger-model.svg)
+**Interledger uses *conditional transfers* to secure payments across multiple hops and even through untrusted connectors.**
+Senders are guaranteed proof that the receiver got the payment or their money back.
+Connectors take some risk, but this risk can be managed and is primarily based upon the connector's chosen ledgers and direct peers.
 
+<span class="show alert alert-info">**Hint:** Conditional transfers or *authorization holds* are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
 
-#### Application Layer
+### Conditional Transfers
 
-The application layer is the top layer of the Interledger protocol suite. Protocols on this layer are responsible for negotiating the key properties of a payment:
+Each local transfer is first *prepared* and then either *executed* or *rejected*.
+When a transfer is prepared, the ledger puts the funds of the source account on hold with a *cryptographic condition* and *timeout*.
+If the condition is fulfilled before the timeout, the transfer is executed and the funds are transferred.
```

Comment:
Again, this should either be bullet points (numbered list?), a separate paragraph with a blank line between it and the previous one, or actually a continuation of the previous line.